.github/workflows/ci.yml

```diff
@@ -25,8 +25,9 @@ jobs:
       run: |
         make -C tools/python metaclean
         make -C tools/python test
-        # Check re-generation didn't change anything (except locale.py)
-        git checkout --  python/phonenumbers/geodata/locale.py
+        # Check re-generation didn't change anything (except locale.py or .pyi files)
+        git checkout -- python/phonenumbers/geodata/locale.py
+        git checkout -- "python/phonenumbers/*data/*.pyi"
```

Comment:
This one confuses me &ndash; why might the `.pyi` files get changed during a CI run?

(`locale.py` can change because `make metaclean` removes it, and its regeneration depends on the local JDK)

Reply:
(#211 now fixed so its output matches the `*data/__init__.pyi` files here)

Reply:
I'll cherry-pick those changes over to incorporate all into the one PR

Reply:
If you've pulled in #211, then this change to `ci.yml` can presumably be dropped?

[Process-wise, I'll probably merge #211 then this PR (once I've finished working through it).]

Reply:
done